www/pages/meeting/[id]/vote.js

```diff
@@ -32,7 +33,27 @@ const Vote = () => {
 
   const loading = !meeting || !topics || !user;
 
+  const userLikes = topics.reduce((acc, t) => {
+    if (t.likes.includes(user.email)) {
+      acc++;
+    }
+
+    return acc;
+  }, 0);
+
+  const maxLikes = Math.floor(topics.length / 2);
```

Comment:
I don't want to send it to the api because then someone could forge it but we could allow max likes to be dynamic on the backend and then return the max likes. I feel like that is something that would fit well on the meeting object if we go down that path. Why do you think we would want it on the group object?